### PR TITLE
realtime update QUEST_LV_0100

### PR DESCRIPTION
**Pull request by terrapin47**
Many corrections to quest text for consistency, grammar, and readability.

Mostly effects the quest line through the mage tower with Grita.

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -675,10 +675,10 @@ QUEST_LV_0100_20150317_000674	Tara Milds
 QUEST_LV_0100_20150317_000675	I am a doctor from Fedimian to help patients.{nl}I have some problems, can you help me?
 QUEST_LV_0100_20150317_000676	There's a sign of plague within those monsters.{nl}Fortunately, I think it's not that serious.{nl}
 QUEST_LV_0100_20150317_000677	Okay.{nl}Let's talk again at Negybas Field.
-QUEST_LV_0100_20150317_000678	You came. Please get some dust sample from Hallowventer.{nl}There are many reasons for a plague to spread.
+QUEST_LV_0100_20150317_000678	You came. Please get some dust samples from the Hallowventers.{nl}There are many reasons that a plague would spread.
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: could? plagues "would"n't do anything imo

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -6715,29 +6715,29 @@ QUEST_LV_0100_20150428_006714	If you are a true Revelator, you can find the divi
...
-QUEST_LV_0100_20150428_006718	We have to be careful from here.{nl}This is the place where Antares, a dangerous magician, was prisoned.{nl}We should move up to the next floor quickly.
+QUEST_LV_0100_20150428_006718	We have to be careful here.{nl}This is the place where Antares, a dangerous magician, was imprisoned.{nl}We should move up to the next floor quickly.
 QUEST_LV_0100_20150428_006719	He was a great magician before.{nl}But, his ideology was too dangerous.
-QUEST_LV_0100_20150428_006720	The reason why he was prisoned here was because he badly wanted the concentrated demons' power here.{nl}What do you think would happen to the tower when the power which was concentrated for hundreds of years explode at once?
+QUEST_LV_0100_20150428_006720	The reason why he was imprisoned here was because he strongly desired the concentrated demon's power here.{nl}What do you think would happen to the tower when the power which was concentrated for hundreds of years exploded all at once?
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: What do you think would happen to the tower if hundreds of years of concentrated power exploded all at once?

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -6715,29 +6715,29 @@ QUEST_LV_0100_20150428_006714	If you are a true Revelator, you can find the divi
...
 QUEST_LV_0100_20150428_006721	Gabija is important, but the tower is more dangerous.{nl}Antares may be engaged in something by now.
 QUEST_LV_0100_20150428_006722	There are two valves here that control the power device.{nl}The first valve is at the lab room so let's go and destroy it now.
 QUEST_LV_0100_20150428_006723	{memo X}Explosion, that was too dangerous.{nl}Let's find a safer way to destroy it.
-QUEST_LV_0100_20150428_006724	Antares was researching for a long time on the flow of the demon's power.{nl}If his research materials are still present, they will help us in destroying those valves.{nl}They must be here somewhere.
+QUEST_LV_0100_20150428_006724	Antares was long researching the flow of the demon's power.{nl}If his research materials are still present, they will help us in destroying those valves.{nl}They must be here somewhere.
```
Comment (vtange) [inline QUEST_LV_0100.tsv]: Antares researched the flow of the demons' power for a long time.{nl}If his research materials are still present, they would help us...

**Comment by terrapin47** [comment]
Resolved conflicts, updated as per feedback.

**Comment by terrapin47** [comment]
This should be good to merge :+1:

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**